tests/test_shapes.py

```diff
@@ -77,25 +77,25 @@ def check_shape_class(self, shape_class):
             if not self.__class__.__name__ == "POSShapeTest":
                 self.assertEqual(shapedef['A'].orientable,
                                  shape_class['params']['orientable'])
-                self.assertEqual(shape_dict['orientable'],
+                self.assertEqual(type_shape['orientable'],
```

Comment:
You should update the `generate_shape_data.py` script to use `dump_shape` instead of `dump_state` for GSD shape definitions and regenerate the files. That way we can get rid of this whole `if 'orientable' in shape_class['params']:` block. I can add state specific tests in a different PR.

Reply:
I updated tests for HOOMD 2.7 (new `get_type_shapes()` and GSD shape support) and included both GSD state and GSD shape tests.

Reply:
I also dropped orientations from the tests.

Reply:
Thanks!